src/server/DRL_parameters.py

```diff
@@ -0,0 +1,25 @@
+class DRL_parameters:
+    def __init__(self,
+        nb_epochs: int,
+        learning_rate: float,
+        batch_size: int,
+        momentum: float) :
+        self.__nbEpochs = nb_epochs
+        self.__learningRate = learning_rate
+        self.__batchSize = batch_size
+        self.__momentum = momentum
+
+    def get_parameters(self):
```

Comment:
Why not replacing this method with the built-in magic method `__dict__` ? Is there a specific implementation purpose for this ?